### ISSUE TITLE
GAN + Pytorch Data Loader relatedQuestion Train and Test Time different data loader methdo use the quality of image is not same.

### ISSUE THREAD

**Issue by praveenkumarchandaliya** [pytorch/vision#1403]
Training Time :  GAN base model I have train using **PyTorch data loader Directory(class)** structure use. Out put is save using vutil.save_grid(-,-,-) method of PyTorch lib. and save result of generated faces in  grid(8*5). or 40 face images in (8*5).  Generated Quality of images is very good.

Test Time :  I am **Not using  PyTorch data loader Directory(class)**  concept. I have load image using Batch wise.
for batch in range(num_batches):
    batch_image = images[batch*batch_size:(batch+1)*batch_size,:,:,:].cuda()
    file_batch = file_list[batch*batch_size:(batch+1)*batch_size]
    .
    .
    .
    than result save using 
    **vutils.save_image(fixed_fake.data,outputpath , normalize=True)**
    But the image Quality go blur or not quality.  If Same model I have test using PyTorch data library loader than image quality is good.

**Comment by pmeier** [comment]
I'm not sure if this is different than #1395. Lets fix the other issue first and see if it resolves this as well.

**Comment by fmassa** [comment]
Closing as this seems stale and it's not clear what the problem is.
